molecule/resources/converge.yml

```diff
@@ -6,6 +6,11 @@
 
   vars:
     openwisp2_network_topology: true
+    # Remove when openwisp-controller==0.8 is released
+    openwisp2_controller_pip: https://github.com/openwisp/openwisp-controller/tarball/master
+    openwisp2_extra_python_packages:
+      - django-flat-json-widget
+      - https://github.com/openwisp/openwisp-notifications/tarball/master
```

Comment:
This shouldn't be needed anymore, we should set ow-controller to 0.8 here: https://github.com/openwisp/ansible-openwisp2/blob/master/defaults/main.yml#L4 and network-topology to 0.5